code/main/server_main.cpp

```diff
@@ -459,7 +459,7 @@ httpd_handle_t start_webserver(void)
     config.server_port = 80;
     config.ctrl_port = 32768;
     config.max_open_sockets = 5; //20210921 --> previously 7   
-    config.max_uri_handlers = 39; // previously 24, 20220511: 35, 20221220: 37, 2023-01-02:38             
+    config.max_uri_handlers = 40; // FIXME: Why not set to a very large value in the first place?
```

Comment:
Because this has an impact on the memory usage

Reply:
There is some (IMHO: weak) evidence, that this value impacts memory usage:

`config.max_uri_handlers = 40;`|`config.max_uri_handlers = 1040;`
---|---
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/6277203/68b43264-e6ec-43cc-836d-62771705d98f)|![image](https://github.com/jomjol/AI-on-the-edge-device/assets/6277203/300b6e17-ff97-4c92-917b-52f1ae46f56e)

I would consider the ~6 Bytes per increment insignificant but we should maybe give some useful info in the comment or remove it altogether.